python/ray/serve/scripts.py

```diff
@@ -122,7 +122,7 @@ def start(
 @click.option(
     "--address",
     "-a",
-    default=os.environ.get("RAY_ADDRESS", "http://localhost:8265"),
+    default=os.environ.get("RAY_AGENT_ADDRESS", "http://localhost:52365"),
```

Comment:
Can we let `serve start` and `serve run` continue to use the `RAY_ADDRESS`? Since they use ray client, they don't need the `RAY_AGENT_ADDRESS`.

Reply:
Yes, start and run are still using RAY_ADDRESS